### PR TITLE
Add sprint report

### PR DESCRIPTION
**Pull request by tubaman**
We are already returning some data from the sprint report.  Create a
generic sprint report method.  Then use then generic sprint report
method in the more granular sprint report data methods.

### CODE REVIEW

File: jira/client.py [hunk 1]
```diff
@@ -4062,35 +4062,35 @@ def update_sprint(self, id, name=None, startDate=None, endDate=None, state=None)
...
+    def sprint_report(self, board_id, sprint_id):
+        """Return the sprint report."""
         return self._get_json(
             "rapid/charts/sprintreport?rapidViewId=%s&sprintId=%s"
             % (board_id, sprint_id),
             base=self.AGILE_BASE_URL,
-        )["contents"]["incompletedIssuesEstimateSum"]["value"]
+        )["contents"]
+
+    def incompletedIssuesEstimateSum(self, board_id, sprint_id):
```
Comment (lan17) [inline jira/client.py]: Just a quick question.  how come the code is mixing camel case and snake case for method names?  AFAIK python convention is to use snake case for the method names.

Comment (ryannowarm) [reply jira/client.py]: Not sure.  I'm just trying to stick to the current convention in `client.py`

**Comment by studioj** [comment]
@tubaman please merge in master and resolve conflicts

**Comment by studioj** [comment]
no action for 3 yrs => close

**Pull request closed without merging**